i18n/zh-Hans/docusaurus-plugin-content-docs/current/architecture/harvesters.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 收割机
-slug: /harvester-architecture
+slug: 收割机架构
```

Comment:
These need to be reverted back to english, I will work on this later in the week

Reply:
these are all fixed now